content/about/mirrors.md

```diff
@@ -54,4 +54,4 @@ The main site hosted at OSGeo (grass.osgeo.org) can be mirrored with the ["rsync
   * `/usr/bin/rsync -az --port=50026 grass.osgeo.org::grass-website /var/www/html/mirror/grass-website`
```

Comment:
Convert to fenced code block.

Reply:
Basically, the bulleted code should just be fenced code blocks. This will make the page look a lot nicer.

Reply:
done